### DISCUSSION TITLE
S2明细表列头分组后导出的格式和表格的格式不一致，第一行会被挤到子节点前一列

### DISCUSSION THREAD

**Discussion by Zhaohanqi12** [antvis/S2#2665] [General]
表格的样式
![image](https://github.com/antvis/S2/assets/166707652/9106d0da-ccd7-4f49-9e79-b295b1149b3b)
导出的样式
![image](https://github.com/antvis/S2/assets/166707652/2c711691-6037-4015-8552-a12a143b6231)

**Comment by lijinke666** [comment]
官网能复现吗? 如不能请提供一个 [codesandbox](https://codesandbox.io/s/29zle) 可复现 Demo

**Reply by Zhaohanqi12** [reply]
2.0版本官网自带的导出方法就有BUG
https://s2.antv.antgroup.com/examples/interaction/basic/#copy-export
自定义导出自定义分组的明细表，分组列不能设置在子节点的上面。

**Reply by Zhaohanqi12** [reply]
https://codesandbox.io/p/sandbox/flamboyant-greider-z9zhfg?layout=%257B%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522rootPanelGroup%2522%253A%257B%2522direction%2522%253A%2522horizontal%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522id%2522%253A%2522ROOT_LAYOUT%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522UNKNOWN%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522clv60mghm0006356fa4tre6sm%2522%252C%2522sizes%2522%253A%255B100%252C0%255D%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522EDITOR%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522EDITOR%2522%252C%2522id%2522%253A%2522clv60mghm0002356fa5ie1obs%2522%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522direction%2522%253A%2522horizontal%2522%252C%2522id%2522%253A%2522SHELLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522SHELLS%2522%252C%2522id%2522%253A%2522clv60mghm0003356f7n5wcapz%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%257D%252C%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522contentType%2522%253A%2522DEVTOOLS%2522%252C%2522id%2522%253A%2522clv60mghm0005356fbn7wcbn2%2522%257D%255D%252C%2522sizes%2522%253A%255B100%255D%257D%255D%252C%2522sizes%2522%253A%255B50%252C50%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clv60mghm0002356fa5ie1obs%2522%253A%257B%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clv60mghm0001356f2s364nkk%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522type%2522%253A%2522FILE%2522%252C%2522filepath%2522%253A%2522%252Fsrc%252Findex.ts%2522%252C%2522state%2522%253A%2522IDLE%2522%257D%255D%252C%2522id%2522%253A%2522clv60mghm0002356fa5ie1obs%2522%252C%2522activeTabId%2522%253A%2522clv60mghm0001356f2s364nkk%2522%257D%252C%2522clv60mghm0005356fbn7wcbn2%2522%253A%257B%2522id%2522%253A%2522clv60mghm0005356fbn7wcbn2%2522%252C%2522tabs%2522%253A%255B%257B%2522type%2522%253A%2522UNASSIGNED_PORT%2522%252C%2522port%2522%253A0%252C%2522id%2522%253A%2522clv612h6w00ty356f2qxv8zkd%2522%252C%2522mode%2522%253A%2522permanent%2522%252C%2522path%2522%253A%2522%252F%2522%257D%255D%252C%2522activeTabId%2522%253A%2522clv612h6w00ty356f2qxv8zkd%2522%257D%252C%2522clv60mghm0003356f7n5wcapz%2522%253A%257B%2522tabs%2522%253A%255B%255D%252C%2522id%2522%253A%2522clv60mghm0003356f7n5wcapz%2522%257D%257D%252C%2522showDevtools%2522%253Atrue%252C%2522showShells%2522%253Afalse%252C%2522showSidebar%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%257D

**Comment by lijinke666** [comment]
重复提问 https://github.com/antvis/S2/issues/2664

**Reply by Zhaohanqi12** [reply]
我看到解决了，但是本地怎么同步这个bug啊

**Reply by lijinke666** [reply]
> 我看到解决了，但是本地怎么同步这个bug啊

周5发布
